style.css

```diff
@@ -75,7 +75,12 @@ a {
     color: var(--card-text);
 }
 
-.repo-details-style {
+.repo-details__display {
+    display: grid;
+    grid-template-columns: 2fr 1fr 1fr 1fr;
+}
+
+.repo-details-font {
```

Comment:
This is weird as a name. Instead I would create a `text-sm` class: https://tailwindcss.com/docs/font-size